File: python/bmi.py
# This python script will calculate the Body Mass Index of an individual

mass = input('Enter your mass in pounds = ')
print('enter your height (feet and inches seperatly)')
feet = input('feet = ')
inches = input('inches = ')

mass = int(mass)
feet = int(feet)
inches = int(inches)

height = (feet*0.3048) + (inches*0.0254)
weight = (mass*0.454)

#need to try (height**2)
bmi = weight/(height*height)

if bmi < 18:
    print("Your BMI is %3.2f, therefore you are underweight" %bmi)
elif bmi >= 18 and bmi < 25:
    print("Your BMI is %3.2f, therefore you have a healthy weigh" %bmi)
elif bmi >= 25 and bmi < 30:
    print("Your BMI is %3.2f, therefore you are a little overweight" %bmi)
elif bmi >= 30:
    print("\nYour BMI is %3.2f, therefore you are obese, hit the gym" %bmi)
else:
    print("there is something wrong with this program")

File: c/bmi.c

#include <stdio.h>

int mass, weight, height;
float bmi;
int feet, inch;

int cal() {
    int h1, h2;
    float result;

    h1 = feet*0.3048;
    h2 = inch*0.0254;

    height = h1 + h2;
    weight = mass*0.454;
    result = weight/(height^2);

    return result;
}

int main() {
    printf("Enter you weight in Pounds: \n");
    scanf("%d", &mass);
    printf("\nEnter your height.\n");
    printf("Feet: ");
    scanf("%d", &feet);
    printf("Inches: ");
    scanf("%d", &inch);
    bmi = cal(feet, inch, mass);

    if (bmi < 18) {
        printf("\nYour BMI is %0.2f, therefore you are underweight.\n", bmi);
    }

    else if (bmi >= 18 && bmi < 25) {
        printf("\nYour BMI is %0.2f, therefore you have a healthy weight.\n", bmi);
    }

    else if (bmi >= 25 && bmi < 30) {
        printf("\nYour BMI is %0.2f, therefore you are a little overweight.\n", bmi);
    }

    else if (bmi >= 30) {
        printf("\nYour BMI is %0.2f, therefore you are obese, hit the gym\n", bmi);
    }

    else {
        printf("\nthere is something with this program\n");
        printf("\nthe BMI is %0.2f\n", bmi);
    }

    return 0;
}
